lib/cash-out/cash-out-atomic.js

```diff
@@ -122,8 +124,9 @@ function updateCassettes (t, tx) {
     tx.deviceId
   ]
 
-  return t.one(sql, values)
-    .then(r => socket.emit(_.assign(r, {op: 'cassetteUpdate', deviceId: tx.deviceId})))
+  return t.one(sql, values).then(r => {
+    return socket.emit(_.assign(r, {op: 'cassetteUpdate', deviceId: tx.deviceId}))
```

Comment:
{op: 'cassetteUpdate', deviceId: tx.deviceId} could be changed to { op: 'cassetteUpdate', deviceId: tx.deviceId } to follow standard. Confirm with Rafael

Also, isn't the return redundant here?